### PR TITLE
chore: Add missing validations for CustomFees

### LINKED ISSUE #14382: Add extra validations for `TokenFeeScheduleUpdateHandler`
Add validations in `TokenFeeScheduleUpdateHandler`  similar to `TokenCreateHandler` for 
1. Negative values of RoyaltyFee
2. Providing empty fees to a token that already has no custom fees there should be (at least by hedera docs) should fail with `CUSTOM_SCHEDULE_ALREADY_HAS_NO_FEES`  response code.

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Fixes #14382 

Adds additional validations for `TokenFeeScheduleUpdate` custom fees handler

### CODE REVIEW

**Review by derektriley — APPROVED** [review]
LGTM.

File: hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java [hunk 1]
```diff
@@ -159,29 +159,11 @@ public void validateForFeeScheduleUpdate(
...
-                    final var royaltyFee = fee.royaltyFeeOrThrow();
-                    if (royaltyFee.hasFallbackFee()
-                            && royaltyFee.fallbackFeeOrThrow().hasDenominatingTokenId()) {
-                        final var tokenNum = royaltyFee
-                                .fallbackFeeOrThrow()
-                                .denominatingTokenId()
-                                .tokenNum();
-                        final var tokenId =
-                                TokenID.newBuilder().tokenNum(tokenNum).build();
-                        validateExplicitTokenDenomination(collectorId, tokenId, tokenRelationStore, tokenStore);
```
Comment (mhess-swl) [inline hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java:179]: Where did this logic go? Is it already contained in `validateFractionalFeeForFeeScheduleUpdate()`?

Comment (Neeharika-Sompalli) [reply hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java:179]: It is in 
```
case ROYALTY_FEE -> // royalty fee can be only applied to non-fungible unique tokens
                validateRoyaltyFee(tokenType, fee, tokenRelationStore, tokenStore);
```

**Review by tinker-michaelj — APPROVED** [review]
LGTM, ty @Neeharika-Sompalli !

**Review by tinker-michaelj — APPROVED** [review]
LGTM, ty @Neeharika-Sompalli !

**Pull request merged**